### ISSUE TITLE
How to update file from .net framework winform program with HttpClient?

### ISSUE THREAD

**Issue by mainware** [IKende/FastHttpApi#132]
[WebHello20200526.zip](https://github.com/IKende/FastHttpApi/files/4681287/WebHello20200526.zip)

client use HttpClient, MultipartFormDataContent

**Comment by IKende** [comment]
`ContentType` add `boundary`  value
``` csharp
 string boundary = string.Format("WebKitFormBoundary{0}", DateTime.Now.Ticks.ToString("x"));
MultipartFormDataContent content = new MultipartFormDataContent(boundary);
 content.Headers.ContentType = MediaTypeHeaderValue.Parse("multipart/form-data;boundary=" + boundary);
```

**Comment by IKende** [comment]
'multipart/form-data' content header analysis function has some error problems

**Comment by IKende** [comment]
fix in v1.8.2.3
